tests/acceptance/features/webUIProvisioning/groups.feature

```diff
@@ -0,0 +1,78 @@
+@webUI
+Feature: add group
+  As an administrator
+  I want to be able to add, delete and modify groups via the webUI
+  So that I can easily manage groups when user LDAP is enabled
+
+  Background:
+    Given user admin has logged in using the webUI
+    And the administrator has browsed to the users page
+
+  Scenario: Add group
+    When the administrator adds group "simple-group" using the webUI
+    Then the group name "simple-group" should be listed on the webUI
+
+  Scenario: Add group with same name as existing ldap group
+    When the administrator adds group "simple-group" using the webUI
+    Then the group name "simple-group" should be listed on the webUI
+    And a notification should be displayed on the webUI with the text "Error creating group: Group already exists."
+
+  Scenario: Add ldap group with same name as existing database group
+    Given group "db-group" has been created in the database user backend
+    When the administrator imports this ldif data:
+      """
+      dn: cn=db-group,ou=TestGroups,dc=owncloud,dc=com
+      cn: db-group
+      gidnumber: 4700
+      objectclass: top
+      objectclass: posixGroup
+      """
+    And the administrator reloads the users page
+    Then the group name "db-group_2" should be listed on the webUI
+    And group "db-group_2" should exist
+
+  Scenario: delete group
+    Given group "simple group" has been created in the database user backend
+    And the administrator has browsed to the users page
+    When the administrator deletes the group named "simple group" using the webUI and confirms the deletion using the webUI
+    Then the group name "simple group" should not be listed on the webUI
+    When the administrator reloads the users page
+    Then the group name "simple group" should not be listed on the webUI
+    And group "simple group" should not exist
+
+  Scenario: delete ldap defined group
+    When the administrator deletes the group named "grp1" using the webUI and confirms the deletion using the webUI
+    Then dialog should be displayed on the webUI
+    | title                       | content                     |
+    | Unable to delete grp1       | Unable to delete group.     |
+    And group "grp1" should exist
+
+  Scenario: Add database user to database group
+    Given user "db-user" has been created with default attributes in the database user backend
+    And group "db-group" has been created in the database user backend
+    And the administrator has browsed to the users page
+    When the administrator adds user "db-user" to group "db-group" using the webUI
+    Then user "db-user" should exist
+    And user "db-user" should belong to group "db-group"
+
+  @issue-core-25224
```

Comment:
Can you add an example (commented-out) different example scenario step that should pass when the issue is fixed.

Reply:
The issue was that it doesn't give any error while adding user to the group while it should not be possible and it must give some kind of error in the UI. But we don't know what kind of error it will give when it is fixed.